internal/cluster/cluster.go

```diff
@@ -1151,7 +1150,7 @@ func (m *Manager) GenerateAdditionalManifests(ctx context.Context, cluster *comm
 		return errors.Wrap(err, "failed to add chrony manifest")
 	}
 
-	if common.IsSingleNodeCluster(cluster) && m.EnableSingleNodeDnsmasq {
+	if common.IsSingleNodeCluster(cluster) {
```

Comment:
Forgot a call to `!m.isSNODNSMasqEnabled()` here?

Reply:
Fixed by exposing `IsSNODNSMasqEnabled` in `ManifestsGenerator`